jscomp/syntax/ast_compatible.mli

```diff
@@ -119,8 +119,27 @@ val fun_ :
   expression -> 
   expression
 
-val is_arg_label_simple : 
-  arg_label -> bool   
+val opt_label : string ->
+#if OCAML_VERSION =~ ">4.03.0" then
+  Asttypes.arg_label
+#else
+  string
+#end
+
```

Comment:
this can be replaced with `val opt_label : string -> arg_label`, in this module, `arg_label` already take into account compiler versions